### ISSUE TITLE
[Python] DefaultApi parentheses missed in client README sample code 

### ISSUE THREAD

**Issue by omidmt** [swagger-api/swagger-codegen#3975]
##### Description

When generating python client by the online generator, the sample code in README doesn't include postfix parentheses in DefaultApi call, that make error when using that code.

 # create an instance of the API class
api_instance = rira_api.DefaultApi
##### Swagger-codegen version

2.2
##### Swagger declaration file content or url

https://raw.githubusercontent.com/omidmt/rira/0.7.x/api/swagger-api.yaml

##### Command line used for generation

curl -X POST -H "content-type:application/json" -d '{"options": {"packageName": "rira_api"},"swaggerUrl":"https://raw.githubusercontent.com/omidmt/rira/0.7.x/api/swagger-api.yaml"}' http://generator.swagger.io/api/gen/clients/python 

##### Suggest a Fix

The DefaultApi.md in doc is correct

**Comment by wing328** [comment]
@omidmt you mean the code should be fixed as follows instead?
```
# create an instance of the API class
api_instance = rira_api.DefaultApi()
```

**Comment by omidmt** [comment]
Yes, because the code without () doesn't work:
TypeError: unbound method sample_operation() must be called with DefaultApi instance as first argument

**Comment by wing328** [comment]
Right, I'll file a PR to fix it accordingly. Thanks for reporting the issue.

**Comment by wing328** [comment]
Fixed via #3977 

Please pull the latest to give it a try.
